saskatoon/harvest/filters.py

```diff
@@ -36,32 +38,41 @@ class HarvestFilter(filters.FilterSet):
             is_staff=True
         ),
         required=False,
+        lookup_expr="exact",
         help_text="",
     )
 
     trees = filters.ModelChoiceFilter(
         queryset=TreeType.objects.all(),
         label=_("Tree"),
+        lookup_expr="exact",
```

Comment:
`lookup_expr` defaults to "exact", it's not really needed to add them.

Reply:
These `exact` declarations were previously on the `fields` attribute under the `Meta` class. Since I had to remove them for the error mentioned in my comment, I have added them up with the each specific fields. Should I remove them?